cola/themes.py

```diff
@@ -0,0 +1,471 @@
+"""Provides themes generator"""
+from __future__ import absolute_import, division, unicode_literals
+
+from qtpy import QtGui
+
+from .i18n import N_
+from .widgets import defs
+from . import icons
+from . import qtutils
+
+
+class EStylesheet(object):
+    DEFAULT = 1
+    FLAT = 2
+
+
+def normalize(val, min_v=0, max_v=1):
+    if val < min_v:
+        return min_v
+    if val > max_v:
+        return max_v
+    return val
```

Comment:
This does normalize the value in that it makes sure it's between 0 and 1, but it doesn't do any scaling (like, for example, when normalizing a waveform, or normalizing a vector).  What it's doing might be better described as a "clamp" operation.

This could be rewritten like this using float literals to make it clear that the inputs and outputs are float.  It turns out we already have a `clamp()` function in `cola/utils.py` ;-) so maybe this could just be:

```
from . import utils

....

def clamp(value):
    return utils.clamp(value, 0.0, 1.0)
```

I was going to mention that you can just define it as `min(1.0, max(value, 0.0))` instead of doing all the branching and then remembered that we have that exact expression in the utils module:

```
# from utils.py

def clamp(value, lo, hi):
    """Clamp a value to the specified range"""
    return min(hi, max(lo, value))
```